.github/workflows/lint.yml

```
@@ -48,18 +48,25 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Detect Aztec version
+        id: aztec-version
+        run: |
+          AZTEC_VERSION=$(node -p "require('./package.json').config.aztecVersion")
+          echo "version=$AZTEC_VERSION" >> "$GITHUB_OUTPUT"
+          echo "Aztec version is ${{ steps.aztec-version.outputs.version }}" 
```

During this `run`the output is not yet defined. It should be:
```suggestion
          echo "Aztec version is $AZTEC_VERSION" 
```